app/controllers/login_controller.rb

```diff
@@ -11,7 +11,7 @@ def create
       view_context.log_in @user
       redirect_to home_path
     else
-      flash.now[:error] = 'Invalid email/password combination'
+      flash.now[:error] = "Invalid email/password combination"
```

Comment:
I don't know if this is the best thing to change. It's usually best to leave trivial things like single vs. double quotes unchanged, since it will distract reviewers from actual changes.